CIScripts/Build.ps1

```diff
@@ -3,23 +3,47 @@
 . $PSScriptRoot\Common\Init.ps1
 . $PSScriptRoot\Common\Job.ps1
 . $PSScriptRoot\Build\BuildFunctions.ps1
-
-$Repos = @(
-    [Repo]::new($Env:DRIVER_REPO_URL, $Env:DRIVER_BRANCH, "src/github.com/codilime/contrail-windows-docker", "master"),
-    [Repo]::new($Env:TOOLS_REPO_URL, $Env:TOOLS_BRANCH, "tools/build/", "windows"),
-    [Repo]::new($Env:SANDESH_REPO_URL, $Env:SANDESH_BRANCH, "tools/sandesh/", "windows"),
-    [Repo]::new($Env:GENERATEDS_REPO_URL, $Env:GENERATEDS_BRANCH, "tools/generateDS/", "windows"),
-    [Repo]::new($Env:VROUTER_REPO_URL, $Env:VROUTER_BRANCH, "vrouter/", "windows"),
-    [Repo]::new($Env:WINDOWSSTUBS_REPO_URL, $Env:WINDOWSSTUBS_BRANCH, "windows/", "windows"),
-    [Repo]::new($Env:CONTROLLER_REPO_URL, $Env:CONTROLLER_BRANCH, "controller/", "windows3.1")
-)
+. $PSScriptRoot\Build\StagingCI.ps1
+. $PSScriptRoot\Build\ProdCI.ps1
 
 $Job = [Job]::new("Build")
 
+$IsTriggeredByGerrit = Test-Path Env:GERRIT_CHANGE_ID
+if($IsTriggeredByGerrit) {
+    # Build is triggered by Jenkins Gerrit plugin, when someone submits a pull
+    # request to review.opencontrail.org.
+
+    $TriggeredProject = Get-GerritProjectName -ProjectString $ENV:GERRIT_PROJECT
```

Comment:
What about dependent reviews?

Reply:
Since this is mostly based on Jarek's original work, we were thinking about implementing this in subsequent PR. Is that OK?

Reply:
JW-1344